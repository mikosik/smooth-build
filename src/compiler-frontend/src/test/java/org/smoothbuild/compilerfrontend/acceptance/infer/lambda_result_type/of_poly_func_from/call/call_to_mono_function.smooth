Int otherFunc() = 7;
myValue = () -> otherFunc();