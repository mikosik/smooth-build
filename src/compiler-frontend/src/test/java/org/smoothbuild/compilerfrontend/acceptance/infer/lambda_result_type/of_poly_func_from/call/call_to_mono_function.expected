Int otherFunc<>()
  = 7;
()->Int myValue<>
  = <>() -> otherFunc<>()<>;
